.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 .idea
 .vagrant
+dist
```

Comment:
What is the `dist` folder for?

Reply:
Mmm, I keep getting the `dist` folder. Sometimes when I'm compiling with an error, sometimes when not having any compile errors. However, before this PR I did not have anything like this. I'll verify that and take a close look as to what has changed.

Reply:
I have reduced this problem to Webstorm (currently 2018.1.2). I'll investigate further.

Reply:
https://youtrack.jetbrains.com/issue/WEB-32527

for now, let's keep this line in `.gitignore`